.github/workflows/linux-x64-gpu.yml

```diff
@@ -46,7 +46,37 @@ jobs:
         uses: addnab/docker-run-action@v3
         with:
           image: openmmlab/lmdeploy-builder:cuda11.8
-          options: -v ${{ github.workspace }}:/work --cpus=1.8
+          options: -v ${{ github.workspace }}:/work
+          run: |
+            cd /work
+            source /opt/conda/bin/activate
+            conda activate py38
+            mkdir build && cd build
+            bash ../generate.sh make
+            make -j$(nproc) && make install
+
+  cuda-121:
+    runs-on: ubuntu-latest
```

Comment:
we can use the matrix strategy for these two jobs:
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#using-a-matrix-strategy